bayesian/api_v1.py

```diff
@@ -96,6 +97,15 @@ def liveness():
     return jsonify({}), 200
 
 
+@api_v1.route('/generatefile', methods=['POST'])
+def generatefile():
```

Comment:
Please create endpoint by deriving from `ResourceWithSchema` to utilize endpoint handling on API side.

Reply:
@fridex we don't want to expose this API end-point

Reply:
and I have made all the necessary changes in the code.

Reply:
> @fridex we don't want to expose this API end-point

What do you mean by this? It needs to be exposed in order to access it.

Reply:
No, I mean API end-point not to be visible. like /liveness end-point.

Reply:
> No, I mean API end-point not to be visible. like /liveness end-point.

I would say this corresponds to API endpoints which should be properly versioned and we should maintain compatibility between versions. `/liveness` and `/readiness` are here for openshift checks and they shouldn't be exposed outside cluster (they are now though). This endpoint will be exposed outside the cluster so users can reach it.